content/en/docs/distributions/charmed/install-kubeflow.md

```diff
@@ -0,0 +1,108 @@
++++
+title = "Installing Kubeflow"
+description = "Instructions for Kubeflow deployment with Kubeflow Charmed Operators"
+weight = 10
++++
+
+This guide lists the steps necessary to install Kubeflow on any conformant Kubernetes, including AKS, EKS, GKE, Openshift and any kubeadm-deployed cluster,  provided that you have access to it via `kubectl`. 
+
+#### 1. Install the Juju client
+
+On Linux, install `juju` via [snap](https://snapcraft.io/docs/installing-snapd) with the following command:
+
+```bash
+snap install juju --classic
+```
+
+Alternatively,  `brew install juju` on macOS or [download the Windows installer](https://launchpad.net/juju/2.8/2.8.5/+download/juju-setup-2.8.5-signed.exe). 
+
+#### 2. Connect Juju to your Kubernetes cluster
+
+In order to operate workloads in your Kubernetes cluster with Juju, you have to add your cluster to the list of *clouds* in juju via the `add-k8s` command.
+
+If your Kubernetes config file is in the standard location (`~/.kube/config` on Linux), and you only have one cluster, you can simply run:
+
+```bash
+juju add-k8s myk8s
+```
+If your kubectl config file contains multiple clusters, you can specify the appropriate one by name:
+
+```bash
+juju add-k8s myk8s --cluster-name=foo
+```
+Finally, to use a different config file, you can set the `KUBECONFIG` environment variable to point to the relevant file. For example:
+
+```bash
+KUBECONFIG=path/to/file juju add-k8s myk8s
+```
+For more details, see the [Juju documentation](https://juju.is/docs/clouds).
+
+#### 3. Create a controller
+
+To operate workloads on your Kubernetes cluster, Juju uses controllers. You can create a controller with the  `bootstrap`  command:
+
+```bash
+juju bootstrap myk8s my-controller
+```
+
+This command will create a couple of pods under the `my-controller` namespace. You can see your controllers with the `juju controllers` command.
+
+You can read more about controllers in the [Juju documentation](https://juju.is/docs/creating-a-controller).
+
+#### 4. Create a model
+
+A model in Juju is a blank canvas where your operators will be deployed, and it holds a 1:1 relationship with a Kubernetes namespace.
+
+You can create a model and give it a name, e.g. `kubeflow`, with the `add-model` command, and you will also be creating a Kubernetes namespace of the same name:
+
+```bash
+juju add-model kubeflow
+```
+You can list your models with the `juju models` command.
+
+#### 5. Deploy Kubeflow
+
+[note type="caution" status="MIN RESOURCES"]
+The minimum resources to deploy `kubeflow` are - 50Gb of disk, 14Gb of RAM and 2 CPUs - available to your Linux machine or VM. 
```

Comment:
Suggestion:

```suggestion
To deploy `kubeflow`, you'll need at least 50Gb available of disk, 14Gb of RAM, and 2 CPUs available in your machine/VM.
```

(because macOS, Windows)